package.json

```diff
@@ -27,6 +27,7 @@
     "express": "^4.17.1",
     "express-rate-limit": "^5.1.1",
     "express-session": "^1.17.0",
+    "js-base64": "^2.5.2",
```

Comment:
remove this package and use `new Buffer(this.clientId + ":" + this.clientSecret).toString('base64')`